lighthouse-cli/cli-flags.js

```diff
@@ -171,6 +181,24 @@ function getFlags(manualArgv) {
           'For more information on Lighthouse, see https://developers.google.com/web/tools/lighthouse/.')
       .wrap(yargs.terminalWidth())
       .argv;
+
+  // ".middleware" does not exist in this version of yargs, so do some post-processing here.
+  /** @type {(keyof LH.CliFlags)[]} */
```

Comment:
I think we're settling on `Array<keyof LH.CliFlags>` for complex types that aren't just `string[]` or w/e, though if you ask @brendankenny we're *always* settling on `Array<*>` ;)

Reply:
> though if you ask @brendankenny we're always settling on Array<*> ;)

well, yes, because it's an array, not a string, so why would you put that information last? :P. They could have gone `[string]`, but they used that up on tuples